distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -833,23 +849,112 @@ public void testCswQueryByTitle() throws Exception {
 
     @Test
     public void testCswQueryForMetacardXml() throws Exception {
-        String titleQuery = getCswQuery("title",
-                "myTitle",
-                "application/xml",
-                "urn:catalog:metacard");
 
-        given().contentType(ContentType.XML)
-                .body(titleQuery)
-                .when()
-                .post(CSW_PATH.getUrl())
-                .then()
-                .assertThat()
-                .body(hasXPath("/GetRecordsResponse/SearchResults/metacard/@id",
-                        is(metacardIds[GEOJSON_RECORD_INDEX])),
-                        hasXPath("/GetRecordsResponse/SearchResults/@numberOfRecordsReturned",
-                                is("1")),
-                        hasXPath("/GetRecordsResponse/SearchResults/@recordSchema",
-                                is("urn:catalog:metacard")));
+        // The first few bytes of the metacard's base-64 encoded thumbnail image
+        String thumbNailBase64EncSubstring = "/9j/4AAQSkZJRgABAQAAAQABAAD";
```

Comment:
It looks like this string could be a constant as it seems to be the same in each test.